tests/unit/type/stream.tcl

```diff
@@ -53,49 +53,49 @@ start_server {
     tags {"stream external-ok"}
 } {
     test {XADD can add entries into a stream that XRANGE can fetch} {
-        r XADD mystream * item 1 value a
-        r XADD mystream * item 2 value b
-        assert_equal 2 [r XLEN mystream]
-        set items [r XRANGE mystream - +]
+        r XADD mystream{t} * item 1 value a
+        r XADD mystream{t} * item 2 value b
+        assert_equal 2 [r XLEN mystream{t}]
+        set items [r XRANGE mystream{t} - +]
```

Comment:
i didn't look at each and every tag, but why did you tag this test?
i don't think we wanna tag all key names in the tests, just ones that are used in tests that interact with more than one key in the same command.

if you did add many unnecessary tags, i'd rather revert that to avoid unnecessary conflicts.

Reply:
There shouldn't be unnecessary tags (unless something was overlooked). In some cases tags are not needed in a specific test but other tests rely on leftover keys and the names must match.

Reply:
maybe of these stream tests (including the one i commented on), work on just one key.
it looked to me as if you did a search-replace in this file.
regardless if you did that, or if it was indeed necessary, i think we need to try a bit harder to avoid these changes.
if there 's indeed such a dependency across so many tests in this file, maybe it's time to refactor it a bit to break it.
i.e. many of the tests work on just one key, and if there's one or two that does work on two keys, we can modify it to be independent.

Reply:
Few excess tags, some dependencies were fixed as well.